setup.py

```diff
@@ -31,16 +31,22 @@ def get_long_description():
       long_description=get_long_description(),
       long_description_content_type="text/markdown",
       classifiers=[
+            "Development Status :: 4 - Beta",
+            "Intended Audience :: Developers",
             "Programming Language :: Python :: 2",
             "Programming Language :: Python :: 2.4",
             "Programming Language :: Python :: 2.6",
             "Programming Language :: Python :: 2.7",
             "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+            "Topic :: Software Development :: Libraries :: Python Modules",
```

Comment:
This isn't accurate, alt-src isn't intended for use as a library.

Reply:
I saw this being used as a dependency to Pub. So should that be put out as a tool?
If so, should this be categorized as Software Development :: Version Control :: Git OR System :: Software Distribution OR Utilities

Reply:
"Software Development :: Version Control :: Git" makes sense to me, but I don't think you really need to add anything here.  I just didn't want to see it falsely referred to as a library.

Reply:
Removed the Topic.